WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -141,6 +141,10 @@ + (NSArray *)routes
     [FBConfiguration forceSimulatorSoftwareKeyboardPresence];
   }
 
+  if (capabilities[FB_CAP_USE_HID_CLEAR]) {
```

Comment:
it is not really needed to add a capability for it. Settings could also be set upon session startup

Reply:
Thank you! removed the capability in https://github.com/appium/WebDriverAgent/pull/952/commits/35afc8ffbbeae9fcd098a1ae292c18ec9b0ff52f and fixed to use setting instead in https://github.com/appium/WebDriverAgent/pull/952/commits/e89b0f77a76ac2ee56826c79bc7ac09cad99781c